### DISCUSSION TITLE
Is it possible to save the video files to NAS?

### DISCUSSION THREAD

**Discussion by Vizards** [koush/scrypted#306] [Q&A] [answered]
I found a built-in plugin called `HomeKit Secure Video Local Copy`, but I'm running into the following two issues:

1. There is no way to specify the save path of the video file.
2. Is there any way to save all the video records instead of video clips？

**Comment by koush** [comment, marked as answer]
Not yet, there will be a paid NVR product that builds on Scrypted soon though.
You can symlink the hksv video path to a NAS.
